src/MinterGateway.sol

```diff
@@ -794,8 +770,28 @@ contract MinterGateway is IMinterGateway, ContinuousIndexing, ERC712Extended {
 
         if (principalOfMaxAllowedActiveOwedM_ >= principalOfActiveOwedM_) return;
 
+        MinterState storage minterState_ = _minterStates[minter_];
+
+        uint40 penalizeFrom_ = UIntMath.max40(minterState_.updateTimestamp, minterState_.penalizedUntilTimestamp);
```

Comment:
Shouldn't this variable be named `penalizedUntil_` since we return early if `newTimestamp_` is lower than `penalizeFrom_`?

Reply:
We are trying to penalize for underollaterlization from `UIntMath.max40(minterState_.updateTimestamp, minterState_.penalizedUntilTimestamp)` to `newTimestamp_`, so it is in fact `penalizeFrom_`.